TeachingRecordSystem/src/TeachingRecordSystem.Core/DataStore/Postgres/Mappings/IntReturnMapping.cs

```diff
@@ -0,0 +1,13 @@
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using TeachingRecordSystem.Core.DataStore.Postgres.Models;
+
+namespace TeachingRecordSystem.Core.DataStore.Postgres.Mappings;
+
+public class IntReturnMapping : IEntityTypeConfiguration<IntReturn>
```

Comment:
Do we still need this in the current version of EF?

Reply:
done